src/navigation/stack/mainBucketStack.js

```diff
@@ -79,13 +78,14 @@ export default navigator;
 const styles = StyleSheet.create({
   customContainer: {
     flex: 1,
-    backgroundColor: "white" ,
+    backgroundColor: 'white',
+    marginTop: '10%',
```

Comment:
use status bar height. not arbitrary 10%

Reply:
@GregTheGreek done